vite.config.js

```diff
@@ -31,9 +31,7 @@ const css = {
   preprocessorOptions: {
     scss: {
       additionalData: `
-        @use 'sass:math';
-        @use 'sass:color';
-        @import 'src/scss/provider-variables.scss';
+        @use 'src/scss/provider-variables.scss' as *;
```

Comment:
It's possible that we should remove this altogether and break up the variables into separate `@use` namespaces imports.. at least that seems to be what scss wants us to do.. but.. I dunno.. this pattern of having a few global scss vars hasn't hurt us at all.. and the imports would be everywhere...

Reply:
So to implement this we'd need to add `@use src/scss/provider-variables.scss` in every `.scss` file? With relative paths too? Like `@use ../../scss/provider-variables.scss`?

When working on this I thought about doing that. But I couldn't figure out another way besides having the `@use` in a ton of different files.

Reply:
yeah.. well I think that's sass intention in deprecating `@import`.  They want explicit and obvious dependencies.. and for people to only use them when necessary... so they'd probably suggest against using `as *` and rather using `as colors` or something where the sass ends up `color: colors.blue;`   In theory I agree... but in practice.. yeah.. it's so many @use when our global "if you see a $variable, you'll find it in our short list of provider-variables.scss" seems to work pretty well for us.  The `additionalData` was always a little hacky.. but in some ways the more we fight the standard the more hacky our solutions are starting to look..   but as you said.. the alternative is just so much `@use` which doesn't really seem to clarify anything IMO.  So I dunno. sass sucks.